spec/models/like_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  it 'AuthorId is present' do
+    subject.AuthorId = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'PostId is present' do
+    subject.PostId = nil
+    expect(subject).to_not be_valid
+  end
+end
```

Comment:
- [ ] Based on the requirements, you ought to add a unit test for your custom method `update_likes_counter`. Adding unit tests for model methods ensures that you can catch bugs and errors before they reach the end-users and have confidence that your application will work as expected, even as you make changes and add new features. Kindly write unit tests for the method. 👍 💻 

#### Hint: Remember that you automatically call the `update_likes_counter` through the `after_save` callback method. I would advise you to create instances of `Like` models and test if the value of the `likes_counter` is equal to the number of `Like` instances that you created. 👍 💻